examples/fabric-chaincode-example-external/build.gradle

```diff
@@ -0,0 +1,326 @@
+/*
+ * Copyright IBM Corp. 2018 All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+plugins {
+    id 'maven-publish'
+    id 'jacoco'
+    id 'maven'
+    id 'signing'
+}
+
+tasks.withType(org.gradle.api.tasks.testing.Test) {
+    systemProperty 'CORE_CHAINCODE_LOGGING_LEVEL', 'DEBUG'
+}
+
+
+dependencies {
```

Comment:
Is all the content required in this build.gradle? For example I can see the `uploadArchives` task which I wouldn't have expected to see in an example.

Reply:
There are other elements as well in this build.gradle file that are specific to the main build.